### ISSUE TITLE
Can we get the "increasing" level to display current battery percentage?

### ISSUE THREAD

**Issue by roccodev** [derfreimann/pokebattle#2]
So we don't have to go in the Menu every time. Also, can it display even while charging?

**Comment by derfreimann** [comment]
Well, it DOES show the battery level on the lower health bar. While charging it just shows the watch as charging though, because the LED indicates when the Pebble's fully charged anyways. But to be honest, is been a looooong time since I used the watches (or my Pebble at all!), so I may not remember it properly 😅

**Comment by derfreimann** [comment]
well, shitty autocorrect... but I guess you know what I ment...

**Comment by roccodev** [comment]
Yes but I would like to see the battery level in percentage (e.g., level 34 for 34%)

**Comment by derfreimann** [comment]
You've got the code and the tools, be my guest! 😉

**Comment by jjmakin** [comment]
Just seen this. I think the actual underlying api only allows you to see battery status as multiples of 10%
